packages/cli/docs/custom-plugins.md

```diff
@@ -0,0 +1,637 @@
+# Integrating a plugin in the CLI (based on real examples)
```

Comment:
```suggestion
# Integrating a plugin in the CLI
```

While I appreciate you read my suggestions, I only included the parentheses as an internal note. I don't think they belong to the public version (similar in the other file...which has since disappeared o.o).